src/writeProtobuf.ts

```diff
@@ -0,0 +1,97 @@
+import { RosMsgDefinition } from "@foxglove/rosmsg";
+import fs from "fs/promises";
+import path from "path";
+
+const BUILTINS_PROTO = `syntax = "proto3";
+
+package ros.builtins;
+
+message Time {
+  fixed32 sec = 1;
+  fixed32 nsec = 2;
+}
+
+message Duration {
+  fixed32 sec = 1;
+  fixed32 nsec = 2;
+}
+`;
+
+const BUILTIN_TYPE_MAP = new Map([
+  ["time", "ros.builtins.Time"],
```

Comment:
`ros.Time` ? tho I guess for consistency having `builtins` is nice. maybe `ros.primatives.Time` ?